joomla/joomla-cms
# [4.0] Skip multiselect execution for links and buttons

Pull Request for Issue #21464 .

### Summary of Changes
Buttons and links handle already the click event, let's not mix them...
Unfortunately, this doesn's solve the issue as it appears to be a css conflict, but nevertheless still a good improvement for multiselect.js

@ciar4n any clues on #21464?

### Testing Instructions
Go to /administrator/index.php?option=com_menus&view=menus

click on any of the modules dropdown
The row SHOULD NOT be selected!!


### Expected result



### Actual result



### Documentation Changes Required
